lib/packwerk/validators/dependency_validator.rb

```diff
@@ -37,7 +37,7 @@ def check_package_manifest_syntax(configuration)
         package_manifests_settings_for(configuration, "enforce_dependencies").each do |config, setting|
           next if setting.nil?
 
-          unless [TrueClass, FalseClass, "strict"].include?(setting.class)
+          unless [TrueClass, FalseClass].include?(setting.class) || setting == "strict"
```

Comment:
```suggestion
          unless [true, false, "strict"].include?(setting)
```
Can't we just do this instead?